arch/network_tcp.c

```diff
@@ -50,6 +50,7 @@ connection_releaserecvbuffer(UA_Connection *connection,
     UA_ByteString_clear(buf);
 }
 
+#ifdef poll
```

Comment:
Isn't it possible to disable the poll bits of the function by ifdef's, so that only 3 + 4 lines are disabled instead of completely duplicating the function?